typings/custom-global/index.d.ts

```diff
@@ -1,10 +1,14 @@
 /**
- * Generic typings for sass files.
+ * Generic typings for non typescript/javascript.
```

Comment:
Since this is a doc comment can you just leave it as-is and add a separate one for the new module type?

Reply:
TBH I'd prefer to remove it completely - its self explanatory from the code what it does